Apply this to the ZPLUG installation. The first change solves(ignores) a security issue.
While the second change is a performance optimisation.

diff --git a/base/core/core.zsh b/base/core/core.zsh
index 4e2624f..74c8254 100644
--- a/base/core/core.zsh
+++ b/base/core/core.zsh
@@ -153,7 +153,7 @@ __zplug::core::core::prepare()
 
     # Run compinit if zplug comp file hasn't load
     if (( ! $+functions[_zplug] )); then
-        compinit -C -d "$ZPLUG_HOME/zcompdump"
+        compinit -u -C -d "$ZPLUG_HOME/zcompdump"
     fi
 }
 
diff --git a/base/job/polling.zsh b/base/job/polling.zsh
index a9cea49..42993c2 100644
--- a/base/job/polling.zsh
+++ b/base/job/polling.zsh
@@ -8,9 +8,9 @@ __zplug::job::polling::periodic()
         if [[ -o monitor ]]; then
             return 0
         fi
-        if setopt monitor; then
-            __zplug::log::write::info "turn monitor on"
-        fi
+        #if setopt monitor; then
+            #__zplug::log::write::info "turn monitor on"
+        #fi
     fi
 }
 
